sway/commands/split.c

```diff
@@ -0,0 +1,87 @@
+#include <string.h>
+#include <strings.h>
+#include "sway/commands.h"
+#include "sway/tree/container.h"
+#include "sway/tree/layout.h"
+#include "sway/tree/view.h"
+#include "sway/input/input-manager.h"
+#include "sway/input/seat.h"
+#include "log.h"
+
+static struct cmd_results *_do_split(int argc, char **argv, int layout) {
```

Comment:
What's with the leading underscore?

Reply:
I dunno this is kind of janky now that I look at it.

Reply:
Yeah, it's a static function, so you don't need to worry about any name/linker collisions.

I'll often do something similar to this when rewriting a function but reusing a name, just to get compiler errors for the old usages, and then remove it when I'm done.